### DISCUSSION TITLE
问题反馈/需求收集

### DISCUSSION THREAD

**Discussion by extrastu** [it5200/flomoPlus#7] [Q&A]
> 有问题/需求可以在这里反馈呃
### 反馈格式
- 版本：
- 问题：
- 期望：
- 联系方式：

> 均为非必填

**Comment by kid101x** [comment]
- 版本：edge 1.3.0
- 问题：网页批注图片出错

http://flomo.oss-cn-shanghai.aliyuncs.com/resource/flomoplus/plus.svg

This XML file does not appear to have any style information associated with it. The document tree is shown below.

**Reply by extrastu** [reply]
请升级为最新版本，最新版本已经解决

**Reply by kid101x** [reply]
Edge没推送😅我再等等吧。
谢谢。

**Reply by extrastu** [reply]
chrome的最新版本已经上架，edge 的应该还没通过（edge审核巨慢）

**Reply by kid101x** [reply]
哈哈哈😂，好的，我等等吧。

**Reply by kid101x** [reply]
再问两个问题：
1. 网页批注是任何网页都有，还是你在插件里设置了白名单？因为我看某个新闻网页上不显示这个。
2. 右键菜单是不是没有了？

**Reply by extrastu** [reply]
1、修复了之前多个嵌套的问题，所以只要这个网页有嵌套网页我就不显示
2、应该是新版本的浏览器升级导致的(我今天升级了下浏览器，也发现这个问题了，已经修复，还没送审，我在等看大家是不是都有这个问题)

**Reply by extrastu** [reply]
[flomoplus-v1.3.1.zip](https://github.com/it5200/flomoPlugin/files/6676264/flomoplus-v1.3.1.zip)

右键的问题可以使用这个渡过下

**Reply by kid101x** [reply]
好的，等edge审核新版了。👍

**Comment by kid101x** [comment]
版本：Edge 1.3.1
问题：“选择标签”项目内再多选情况下只有第一个标签会起作用，后面的都无效。

**Reply by extrastu** [reply]
好的

**Comment by shaojiaqi** [comment]
版本：1.3.3
问题：希望能把neat reader 读书笔记导入到flomo 
期望：目前使用场景如下，希望能直接把neat reader导入读书笔记到flomo。
- Neat-Reader 类似于微信读书，不过平常使用客户端阅读书籍，其笔记可以在网页输出json数据，不知道能否有比较方便的方法，导入flomo？已查阅flomo help，没有找到现成的工具。
- 目前我正在使用的方法，导出txt文本后，一段段复制到flomo中，还好笔记不多~至于批注，我在打印稿上手写好了的，打算导入到flomo后再整理批注。

**Reply by extrastu** [reply]
好的，我看下
